catroid/src/main/java/org/catrobat/catroid/ui/settingsfragments/SettingsFragment.java

```diff
@@ -130,6 +130,8 @@ public class SettingsFragment extends PreferenceFragment {
 	public static final String TAG = SettingsFragment.class.getSimpleName();
 
 	public static final String SETTINGS_USE_CATBLOCKS = "settings_use_catblocks";
+	public static final String SETTINGS_CATBLOCKS_ADV_MODE = "setting_enable_catblocks_advanced_mode";
```

Comment:
Please don't use abbreviations for variable names according to Catroid coding style.